package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-devtools-installer",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Please don't bump the version of this package, I'll do that when I want to release a new version